app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title = ""
+    base_title = t("app.title")
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
```

Comment:
Đoạn if else này em áp dụng toán tử 3 ngôi nha

Reply:
@anlvq-1058  em sửa xong rồi ạ